packages/ai-bot/tests/prompt-construction-test.ts

```diff
@@ -33,6 +33,19 @@ import {
   SKILL_INSTRUCTIONS_MESSAGE,
 } from '@cardstack/runtime-common/ai';
 
+const DEFAULT_CATALOG_REALM_URL = 'http://localhost:4201/catalog/';
+const catalogRealmURL = ensureTrailingSlash(
+  process.env.RESOLVED_CATALOG_REALM_URL ?? DEFAULT_CATALOG_REALM_URL,
+);
+
+function ensureTrailingSlash(url: string) {
+  return url.endsWith('/') ? url : `${url}/`;
+}
```

Comment:
This is defined six times in this PR, maybe [this could be imported instead](https://github.com/cardstack/boxel/blob/30a17f383af5ec8753b440880f4efdce6ddfa31d/packages/runtime-common/paths.ts#L70-L72)? (And redefined if this version seems more readable)